### PR TITLE
Secure Routes with password

### PR DESCRIPTION
**Pull request by Can-Tina**

### CODE REVIEW

File: src/server/controllers/rating.js [hunk 1]
```diff
@@ -1,9 +1,16 @@
...
 const getRatings = async (req, res) => {
-    const ratings = await prisma.rating.findMany({})
-    res.json({ data: ratings })
+    const {
+        getPassword
+    } = req.body
+    if (getPassword === password) {
+        const ratings = await prisma.rating.findMany({})
+        res.json({ data: ratings })
+    }
```
Comment (TheunisK) [inline src/server/controllers/rating.js:13]: This code looks great. Very Clean. Would you take me through it when you have a minute?

File: src/server/controllers/pokemon.js [hunk 2]
```diff
@@ -13,33 +15,39 @@ const createSinglePoke = (pokemon, arrIndex) => {
...
-    console.log("pokemons created", pokemons)
-    res.json({ data: pokemons})
+        const pokemons = await prisma.pokemon.createMany({
+            data: pokeArr
+        })
+
+        console.log("pokemons created", pokemons)
+        res.json({ data: pokemons })
+    }
+    res.json("No Access")
```
Comment (nathanSamosa) [inline src/server/controllers/pokemon.js:50]: looks great! The next step is error middleware... spooky

Comment (glowkeeper) [reply src/server/controllers/pokemon.js:50]: https://github.com/boolean-uk/auth-request-flow-exemplar/tree/steve/src - that's an example of error middleware ;)

**Comment by glowkeeper** [comment]
Approved

**Pull request merged**